config/services.yaml

```diff
@@ -17,6 +17,7 @@ parameters:
       sessionlearningmaterials,sessiondescriptions,sessionlearningmaterials,sessions,sessiontypes,schoolconfigs,schools,
       terms,usermadereminders,userroles,users,vocabularies
     ilios_api_valid_api_versions: 'v1'
+    sentry_dsn: https://c70286fb157048be9ebc6e918e8c2b79@sentry.io/1323198
```

Comment:
what's the reason for hard-wiring this value?

Reply:
It shouldn't be a DB config item, as then we'd have to pass it out to schools to configure. I was considering a constant in the class, but this way it's a bit reusable.  We have it similarly hardwired in frontend at https://github.com/ilios/frontend/blob/master/config/environment.js#L115

Reply:
fair enough. thanks!